app/controllers/index.py

```diff
@@ -47,19 +51,53 @@ async def communities():
     return render_response('communities.jinja2', {'local_chapters': LOCAL_CHAPTERS})
 
 
+@router.get('/copyright/{locale:str}')
+async def copyright_i18n(locale: str):
+    if not is_valid_locale(locale):
+        return Response(None, status.HTTP_404_NOT_FOUND)
+    with translation_context(primary_translation_language()):
+        copyright_notice_title = t('site.copyright.foreign.title')
+        original_copyright_link_text = t('site.copyright.foreign.english_link')
+        original_copyright_link = f'<a href="/copyright/en">{original_copyright_link_text}</a>'
+        copyright_notice_html = t('site.copyright.foreign.html', english_original_link=original_copyright_link)
```

Comment:
Isn't it better to keep this in the copyright.jinja2 template? CTRL+F for "{% set" to see how to do it.